Resources/Prototypes/Entities/Mobs/Customization/Markings/ears.yml

```diff
@@ -1,10 +1,65 @@
 - type: marking
-  id: HumanLongEars
+  id: LongEarsStandard
```

Comment:
I recommend not changing this ID as it will break any characters already using the marking.

Reply:
> I recommend not changing this ID as it will remove the marking from players already using it, but it's not necessary requirement

I don't think those who use these ears will want to play with them when the best alternative is available.